Documentation/conf.py

```diff
@@ -25,6 +25,8 @@
 sys.path.insert(0, os.path.abspath('_exts'))
 import cilium_spellfilters
 
+# For making external links open in new tabs
```

Comment:
This comment is unnecessary for the import, it's trivial to figure out by reading the other comment you added where we `cilium_external_links`.

Reply:
True....Thanks for the feedback.